src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Queries/GetOrderQueryHandler.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using VirtoCommerce.ExperienceApiModule.Core.Infrastructure;
+using VirtoCommerce.OrdersModule.Core.Model;
+
+namespace VirtoCommerce.ExperienceApiModule.XOrder.Queries
+{
+    public class GetOrderQueryHandler : IQueryHandler<GetOrderQuery, CustomerOrder>
+    {
+        private readonly ICustomerOrderAggregateRepository _customerOrderAggregateRepository;
+        public GetOrderQueryHandler(ICustomerOrderAggregateRepository customerOrderAggregateRepository)
+        {
+            _customerOrderAggregateRepository = customerOrderAggregateRepository;
+        }
+
+        public Task<CustomerOrder> Handle(GetOrderQuery request, CancellationToken cancellationToken)
+        {
```

Comment:
put validator call here

Reply:
why do we need to use the validator for queries?